### ISSUE TITLE
Interpolated variables not enclosed in braces are not noticed

### ISSUE THREAD

**Issue by Sleft** [rubocop/rubocop#59]
When you interpolate global, instance or class variables in strings you can omit the enclosing braces. But RuboCop does not notice variables not enclosed in braces and gives a misleading warnings of

> Prefer single-quoted strings when you don't need string interpolation or special symbols.

as double-quotation is required for the strings in question.

For example, consider the following code:

``` ruby
# -*- coding: utf-8 -*-

class Klass
  def meth
    @bar = @@bar = $bar = 'baz'
    puts "foo#@bar"
    puts "foo#@@bar"
    puts "foo#$bar"
  end
end
```

When inspecting this rubocop says

```
C:  5: Replace class var @@bar with a class instance var.
C:  6: Prefer single-quoted strings when you don't need string interpolation or special symbols.
C:  7: Prefer single-quoted strings when you don't need string interpolation or special symbols.
C:  7: Replace class var @@bar with a class instance var.
C:  8: Prefer single-quoted strings when you don't need string interpolation or special symbols.
```

But replacing the strings with single-quoted ones will not allow for interpolation.
